src/Product/ProductInfo.cs

```diff
@@ -36,5 +36,12 @@ public static string GetDataApiBuilderUserAgent()
     {
         return Environment.GetEnvironmentVariable(DAB_APP_NAME_ENV) ?? DEFAULT_APP_NAME;
     }
+
+    public static string GetDataApiBuilderApplicationName()
```

Comment:
nit: this does more than just return the app name. i.e. returns: `dab_oss_0.10.5` 

a comment to not how this function differs from `GetProductVersion()` would be helpful.

Reply:
Make sense, added